manifests/php.yml

```diff
@@ -20,7 +20,7 @@ tests/:
         Test_Schema_Request_Json_Body: v0.98.1
         Test_Schema_Request_Path_Parameters: missing_feature
         Test_Schema_Request_Query_Parameters: v0.94.0
-        Test_Schema_Response_Body: v0.98.1
+        Test_Schema_Response_Body: v0.99.1
```

Comment:
nit: I guess you could've used `missing_feature` as well? Probably not very important as we'll look into it soon.